__tests__/home.test.js

```diff
@@ -45,13 +45,11 @@ describe('Text Rendering Tests', () => {
     cleanup();
   });
 
-  /* Since Linting, this is going to require a function to do the text matching */
-  /*     test('Website About Text Renders', () => {
-        render(<Home />)
-
-        expect(screen.getByText('Don\'t know what')).toBeInTheDocument();
-        cleanup()
-    }) */
+  test('Website About Text Renders', () => {
```

Comment:
Im not sure what this is actually asserting on. Could you clarify this in the test header or on the expect statement?

Reply:
Let me add a comment to the file & change the test title (it's in reference to the paragraph than renders on landing that explains the app). I'll ping when I commit!

Reply:
Looks good!